th2_data_services/provider/v5/provider_api/http.py

```diff
@@ -208,13 +224,23 @@ def __create_stream_connection(self, url: str):
 
         response.release_conn()
 
-    # TODO - add DocString (Grigory)
-    def execute_sse_request(self, url: str):
+    def execute_sse_request(self, url: str) -> Generator[dict, None, None]:
+        """Creates SSE connection to server.
+        Args:
+            url: Url.
+        Returns:
+            dict: Response data.
+        """
         response = self.__create_stream_connection(url)
         client = SSEClient(response, char_enc=self._char_enc, decode_errors_handler=self._decode_error_handler)
         for record in client.events():
             yield record
 
-    # TODO - add DocString (Grigory)
-    def execute_request(self, url: str):
+    def execute_request(self, url: str) -> Response:
+        """Sends a GET request to provider.
```

Comment:
By style guide https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html we should add `\n` after the first string.